src/libmongoc/tests/mock_server/mock-server.c

```diff
@@ -1160,6 +1135,83 @@ mock_server_receives_query (mock_server_t *server,
    return request;
 }
 
+/*--------------------------------------------------------------------------
+ *
+ * mock_server_receives_hello_op_msg --
+ *
+ *       Pop a client non-streaming hello call if one is enqueued,
+ *       or wait up to request_timeout_ms for the client to send a request.
+ *
+ * Returns:
+ *       A request you must request_destroy, or NULL if the current
+ *       request is not a hello command.
+ *
+ * Side effects:
+ *       Logs if the current request is a hello command using OP_MSG.
+ *
+ *--------------------------------------------------------------------------
+ */
+
+request_t *
+mock_server_receives_hello_op_msg (mock_server_t *server)
+{
+   bson_t *msg =
+      tmp_bson ("{'hello': 1, 'maxAwaitTimeMS': { '$exists': false }}");
+
+   return _mock_server_receives_single_msg (server, 0, msg);
+}
+
+/*--------------------------------------------------------------------------
+ *
+ * mock_server_receives_bulk_msg --
```

Comment:
I suggest moving `mock_server_receives_bulk_msg` up back to the original location to reduce the diff.